ApplicationPattern+profileInstances.yaml

```diff
@@ -43,6 +43,101 @@ profile-instances:
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: '*-response-p-005' ## TODO: Replace "*" by ApplicationID and ReleaseNumber
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      concrete-field-name: 'ownerName'
+      description: 'Name of the ApplicationOwner'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-name'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: '*-response-p-006' ## TODO: Replace "*" by ApplicationID and ReleaseNumber
+    capability:
+      operation-name: '/v1/inform-about-application-in-generic-representation'
+      concrete-field-name: 'ownerEmailAddress'
+      description: 'Email address of the ApplicationOwner'
+      datatype: 'string'
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/owner-email-address'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: '*-response-p-007' ## TODO: Replace "*" by ApplicationID and ReleaseNumber
+    capability:
+      operation-name: '/v1/inform-about-release-history-in-generic-representation'
+      field-name-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/release-number'
+      description: 'Represents the first entry in the list of releases'
+      datatype: string
+    configuration:
+      value-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-bm-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-list=0/changes'
+
+  - profile-name: 'GenericResponseProfile'
+    uuid: '*-response-p-???' ## TODO: Replace "*" by ApplicationID and ReleaseNumber and ??? by SequenceNumber
+    capability:
+      operation-name: ## TODO: Add generic representation operation that shall provide this response
+      concrete-field-name: ## TODO: Add the concrete name of the field as it shall be respresented in the GUI EXOR a reference from which the name of the field shall be taken. ...
+      field-name-reference: ## TODO: ... Delete either the concrete-field-name or the field-name-reference attribute from the Instance of GenericResponseProfile.
+      description: ## TODO: Add some description for your own reference
+      datatype: ## TODO: Add the datatype of the value
+    configuration:
+      concrete-value: ## TODO: Add the current, concrete value of the field EXOR a reference from which the field shall be filled. ...
```

Comment:
A bit nitpicky: concrete also translates to a building material. I think specific-value or static-value would be more descriptive.

Reply:
Changed for static-value